afex_integration/models/account.py

```diff
@@ -515,6 +564,18 @@ def create_afex_trade(self):
                 'afex_stl_invoice_id': inv_head.id,
                 })
 
+            if (payment.afex_direct_debit
+                    and payment.afex_stl_invoice_id.currency_id == payment.aud_currency_id):
```

Comment:
So, this would be
if payment.afex_direct_debit and payment.afex_stl_invoice_id.currency_idayment.afex_direct_debit_journal_id.currency_id (or some function that correctly identifies 100% that t6he dd_journal currency is definitely the same....